src/Components/ReviewContainer.js

```diff
@@ -1,16 +1,15 @@
 import React from 'react';
 import ReviewCard from './ReviewCard';
 
-const ReviewContainer = (clients) => {
-    console.log("reviewContainer", clients.clients)
+const ReviewContainer = ({reviews}) => {
+    console.log("reviewContainer", reviews)
 
-    const allClients = clients.reviews.map(review => {
+    const allReviews = reviews.map(review => {
```

Comment:
It allows you to use the passed in reviews and not create another copy of them? I wonder why this isnt the case for clients.  I want to look into async/await while using graphql